Mail/Helpers/RichTextEditor.swift

```diff
@@ -96,11 +96,29 @@ struct RichTextEditor: UIViewRepresentable {
             }
         }
 
+        @MainActor
         func editorContentChanged(_ editor: SQTextEditorView, content: String) {
-            var parentBody = parent.body.trimmingCharacters(in: .whitespacesAndNewlines)
-            parentBody = parentBody.replacingOccurrences(of: "\r", with: "")
-            if parentBody != content {
-                parent.body = content
+            let parentBody = parent.body
+            Task.detached {
+                var parentBody = parentBody
+                parentBody = parentBody.trimmingCharacters(in: .whitespacesAndNewlines)
+                parentBody = parentBody.replacingOccurrences(of: "\r", with: "")
+                
+                // Remove explicit dark text css and caret for simple darkmode support.
+                parentBody = parentBody.replacingOccurrences(of: "caret-color: rgb(0, 0, 0);", with: "")
```

Comment:
This works, but could be more aesthetic with a regex to handle the ";"

Reply:
The comment above line 108 seems outdated ?